### PR TITLE
[6.x] Core/Scripts: Monk Spells

### PR DESCRIPTION
**Pull request by Infamous-devel**
**Changes proposed**:
- Monk Spells Scripts

**Target branch**: 6x

**Known issues and TODO list**:
- [ ] Few Spells mostly commented. (Can't fix errors with API).

### CODE REVIEW

File: sql/updates/world/6.x/2016_04_22_00.sql [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+INSERT INTO spell_script_names (spell_id,ScriptName) values (124081,'spell_monk_zen_sphere');
+INSERT INTO spell_script_names (spell_id,ScriptName) values (123986,'spell_monk_chi_burst');
+INSERT INTO spell_script_names (spell_id,ScriptName) values (115288,'spell_monk_energizing_brew');
+INSERT INTO spell_script_names (spell_id,ScriptName) values (116705,'spell_monk_spear_hand_strike');
+INSERT INTO spell_script_names (spell_id,ScriptName) values (116841,'spell_monk_tigers_lust');
+INSERT INTO spell_script_names (spell_id,ScriptName) values (115057,'spell_monk_flying_serpent_kick');
+INSERT INTO spell_script_names (spell_id,ScriptName) values (115008,'spell_monk_chi_torpedo');
+INSERT INTO spell_script_names (spell_id,ScriptName) values (121828,'spell_monk_chi_torpedo');
+INSERT INTO spell_script_names (spell_id,ScriptName) values (119582,'spell_monk_purifying_brew');
+INSERT INTO spell_script_names (spell_id,ScriptName) values (122057,'spell_monk_clash');
```
Comment (Shauren) [inline sql/updates/world/6.x/2016_04_22_00.sql]: These scripts are severely outdates - Clash was removed in WoD

**Comment by digz6666** [comment]
And they will say do it one by one :)

**Comment by MitchesD** [comment]
you should divide it into more PRs at least, it's too much huge diff, github doesn't show it, we can't comment it

File: src/server/scripts/Spells/spell_script_loader.cpp [hunk 2]
```diff
@@ -1,52 +1,54 @@
-/*
```
Comment (Aokromes) [inline src/server/scripts/Spells/spell_script_loader.cpp]: revert this change.

**Comment by Infamous-devel** [comment]
Will create smaller PR for each Spell.
